examples/README.md

```diff
@@ -155,8 +155,8 @@ Example | File | Description
 
 Example | File | Description
 --- | --- | ---
-`ambiguity_checker` | [`ecs/ambiguity_checker.rs`](./ecs/ambiguity_checker.rs) | Shows how the ambiguity checker works
 `ecs_guide` | [`ecs/ecs_guide.rs`](./ecs/ecs_guide.rs) | Full guide to Bevy's ECS
+`ambiguity_checker` | [`ecs/ambiguity_checker.rs`](./ecs/ambiguity_checker.rs) | Shows how the ambiguity checker works
```

Comment:
```suggestion
`ambiguity_checker` | [`ecs/ambiguity_checker.rs`](./ecs/ambiguity_checker.rs) | Shows how to use system order ambiguity checker
```

Reply:
Done in https://github.com/bevyengine/bevy/pull/2766/commits/ab5dd30af37f4f955af3fe0ff0768f507f903272